packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -105,7 +100,7 @@ export class ZipkinExporter implements SpanExporter {
   private _toZipkinSpan(span: ReadableSpan): zipkinTypes.Span {
     return toZipkinSpan(
       span,
-      this._serviceName,
+      String(this._serviceName),
```

Comment:
While this technically also solves the typing problem, it is also non-obvious what is happening and causes an extra allocation of a string object on every span which has a nontrivial performance impact. I still think inlining `_toZipkinSpan` is the right way to go.

Reply:
Do you mean something like this?
```
const zipkinSpans = spans.map(span => toZipkinSpan(
      span,
      this._serviceName,
      this._statusCodeTagName,
      this._statusDescriptionTagName
));
```
I've tried the above code and I still get typing errors.

Reply:
That is because the map happens within sendSpans, not export. That means the typing is still lost. You can move the convert out of `sendSpans` since it isn't technically part of the sending process anyways that might make more sense.

Reply:
Even when I move the map to export, I get the type error. I think it's because the call to `toZipkinSpan` is done in the callback of the map and the typings are lost again.

Reply:
ah yes. rats. just pass the servicename directly to sendSpans i guess